packages/shared/src/node/env/ConsoleLog.ts

```diff
@@ -1,26 +1,33 @@
 import chalk from 'chalk';
 
 export default class ConsoleLog {
+  static prefixes = Object.freeze({
+    info: chalk.bold(chalk.blue('info')),
+    warn: chalk.bold(chalk.yellow('warn')),
+    error: chalk.bold(chalk.red('error')),
+    fatal: chalk.bold(chalk.red('fatal')),
+  });
+
   plain = console.log;
 
   info: typeof console.log = (...args) => {
-    console.log(chalk.bold(chalk.blue('info')), ...args);
+    console.log(ConsoleLog.prefixes.info, ...args);
   };
```

Comment:
nit: define `prefixes` as a private member and use `this.prefixes.info` to access its value?

Reply:
thought about this, but it seems not needed until we allow defining custom prefixes for each instance